plotsearch/models/plot_search.py

```diff
@@ -566,6 +567,14 @@ def lessor_name(self):
         blank=True,
     )
 
+    # GDPR API
+    serialize_fields = (
+        {"name": "address"},
+        {"name": "received_date"},
+        # plotsearch.AreaSearchAttachment
```

Comment:
I would append this comment to the next line, but a matter of taste.

Reply:
Unless the formatter forces it this way

Reply:
Same thought for comments in `users/models.py`

Reply:
I don't exactly remember, but I think there could potentially be cases where there would be multiple fields defined for one model. I thought this way of commenting would then make more sense than duplicating the same comment on two lines.